tests/test_accounts.py

```diff
@@ -0,0 +1,120 @@
+"""
+Tests the GET and POST functionality of our api.
+"""
+import ast
+from typing import Tuple
+
+import pytest
+from flask import request
+from requests import PreparedRequest
+
+
+@pytest.mark.usefixtures("client_class")
+class TestAccounts:
+    def test_new_post(self):
+        """test_new_post Creates a random email address and posts it
+         to the account store,then checks that the email address
+        exists in the account store.
+
+        GIVEN The flask test client
+        WHEN an email is submitted
+        THEN we expect that the email exists in the account store.
+
+        """
+
+        status_code, response_data = self.post_email_and_return_data(
+            "test@delete_me.org"
+        )
+
+        assert status_code == 200
+        assert "account_id" in response_data.keys()
+        assert "applications" in response_data.keys()
+        assert response_data["email_address"] == "test@delete_me.org"
+
+    def test_double_post_returns_409(self):
+
+        params = {"email_address": "test2@delete_me.org"}
+
+        req = PreparedRequest()
+        root_url = request.root_url
+        url = root_url + "account"
+        req.prepare_url(url, params)
+
+        response1 = self.client.post(req.url)
+        response2 = self.client.post(req.url)
+
+        assert response1.status_code == 200
+        assert response2.status_code == 409
+
+    def test_get_methods_work(self):
+
```

Comment:
Maybe some comments here could be handy - perhaps a little hard to follow what is being tested exactly (or why they should be returning what they are). Alternatively perhaps slightly more descriptive naming of returned values (rather than just email...account_id)?